apps/emqx_gateway/src/emqx_gateway_insta_sup.erl

```diff
@@ -197,52 +205,102 @@ detailed_gateway_info(State) ->
 %% Internal funcs
 %%--------------------------------------------------------------------
 
-do_init_authn(GwName, Config) ->
-    case maps:get(authentication, Config, #{enable => false}) of
-        #{enable := false} -> undefined;
-        AuthCfg when is_map(AuthCfg) ->
-            case maps:get(enable, AuthCfg, true) of
-                false ->
-                    undefined;
-                _ ->
-                    %% TODO: Implement Authentication
-                    GwName
-                    %case emqx_authn:create_chain(#{id => ChainId}) of
-                    %    {ok, _ChainInfo} ->
-                    %        case emqx_authn:create_authenticator(ChainId, AuthCfg) of
-                    %            {ok, _} -> ChainId;
-                    %            {error, Reason} ->
-                    %                ?LOG(error, "Failed to create authentication ~p", [Reason]),
-                    %                throw({bad_authentication, Reason})
-                    %        end;
-                    %    {error, Reason} ->
-                    %        ?LOG(error, "Failed to create authentication chain: ~p", [Reason]),
-                    %        throw({bad_chain, {ChainId, Reason}})
-                    %end.
+%% same with emqx_authentication:global_chain/1
```

Comment:
export emqx_authentication:global_chain/1 please and use it in here

Reply:
I'll improve it in the later PR